x-pack/plugins/canvas/scripts/clean_storybook_dll.js

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+const del = require('del');
+const { DLL_OUTPUT } = require('../.storybook/constants');
+
+console.log('cleaning', DLL_OUTPUT);
```

Comment:
nit: should this be using a `log` object?

Reply:
Feels a little like overkill, but if you want to:

```
const { ToolingLog } = require('@kbn/dev-util')
const log = new ToolingLog({ level: 'info', writeTo: process.stdout })
```

Reply:
@spalger Just curious about best practices, and how it would look in the output, that's all. :-)

Reply:
I actually just removed the console log, we log the script executed as part of kbn-pm which should be sufficient:

![image](https://user-images.githubusercontent.com/40265/56315813-9132b780-610d-11e9-96e4-98149eef94f6.png)

Reply:
> Just curious about best practices